fli-iam/shanoir-ng
# shanoir-issue#2161: fix copy of datasets

How to test:
Create new study
Copy existing datasets in the new study
Check that it worked properly
Try to delete the copied examination

Before: an error was thrown, saying there is an issue with mr_protocol
Now: the examination is deleted
